.travis.yml

```diff
@@ -27,6 +27,7 @@ matrix:
 
     # run cypress tests
       node_js: 10
+      if: (type = pull_request AND branch = master) OR commit_message =~ /#travis/
```

Comment:
This means: Run cypress after any pull request to master. We wanted to avoid it, didn't we?

Reply:
I probably misunderstood that. So we do not want to run cypress tests during any pull request, only on commits in master branch (which are mostly automatically created after merging PR)?

Reply:
exactly